site/content/en/docs/tasks/run/argo_workload.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Run An Argo Workflow"
+date: 2024-01-03
+weight: 7
+description: >
+  Integrate Kueue with Argo Workflows.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [Argo Workflows](https://argo-workflows.readthedocs.io/en/latest/).
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
+
+As of the writing of this doc, Kueue doesn't support Argo Workflows [Workflow](https://argo-workflows.readthedocs.io/en/latest/workflow-concepts/) resources directly,
+but you can take advantage of the ability for Kueue to [manage plain pods](/docs/tasks/run_plain_pods) to integrate them.
+
+1. By default, the integration for `v1/pod` is not enabled.
+   Learn how to [install Kueue with a custom manager configuration](/docs/installation/#install-a-custom-configured-released-version)
+   and enable the `pod` integration.
+
+   The pod integration for Argo Workflows looks like:
+   ```yaml
+   apiVersion: config.kueue.x-k8s.io/v1beta1
+   kind: Configuration
+   integrations:
+     frameworks:
+      - "pod"
+     podOptions:
```

Comment:
Instead of podOptions we are migrating to use `managedJobsNamespaceSelector` as a more generic solution, also for plain pods. Could you update as currently in https://kueue.sigs.k8s.io/docs/tasks/run/plain_pods/

cc @dgrove-oss

Reply:
I'll just link to that section.